src/fundus/scraping/crawler.py

```diff
@@ -371,19 +383,34 @@ def __init__(
         self.start = start
         self.end = end
         self.processes = os.cpu_count() or 0 if processes == -1 else processes
+        self.retries = retries
         self.server_address = server_address
 
-    @staticmethod
     def _fetch_articles(
+        self,
         warc_path: str,
         publishers: Tuple[PublisherEnum, ...],
         error_handling: Literal["suppress", "catch", "raise"],
         extraction_filter: Optional[ExtractionFilter] = None,
         url_filter: Optional[URLFilter] = None,
     ) -> Iterator[Article]:
-        source = CCNewsSource(*publishers, warc_path=warc_path)
-        scraper = CCNewsScraper(source)
-        yield from scraper.scrape(error_handling, extraction_filter, url_filter)
+        retries: int = 0
+        while True and retries <= self.retries:
```

Comment:
Doesn't this simplify to `retries <= self.retries`?

Reply:
That's `True` *slapstick drum sound*